flair/trie/replication.go

```diff
@@ -53,25 +66,29 @@ func (r *Replicator) Sequential(key string, f ReplicatedFunc) error {
 // because they're all inline.
 func (r *Replicator) SequentialAck(key string, f ReplicatedAckFunc) error {
 	var e error
+	success := false
 	offset := 0
 	for i := 0; i < r.Replicas; i++ {
-		shouldContinue, err := f(r.Trie.GetOffset(key, offset))
+		shouldContinue, err := r.callAck(f, r.Trie.GetOffset(key, offset))
 		if !r.shouldRetry(err) && !shouldContinue {
```

Comment:
Might read better if we rename shouldRetry to isContinueable. 
```
!(r.isContinuable(err) && shouldConinue() ) {
   return err
}
```

This also leaves us the opportunity to introduce the idea of errors we can retry on the same replica.

Reply:
Yeah that is a better name, agreed.